6.0/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6.0
+FROM node:6.2
```

Comment:
Yea, you'd want to change the folder as well, and then we would need to edit the docker hub triggered builds:

![image](https://cloud.githubusercontent.com/assets/145406/16693049/444e6f70-44e8-11e6-90ee-53f5cf4faba3.png)

We can totally do this, I'm up for this change. Its really on you guys about how comfortable you are versus going `node:6` (6.*.*) or `node:6.2` (6.2.*).

If you need access to the docker hub, let me know.  We may want to keep the 6.0 image around so that any projects that don't update are kept in sync with upstream changes, unless you plan to swap those all right away?

![image](https://cloud.githubusercontent.com/assets/145406/16693108/82852b94-44e8-11e6-8bff-d2c3670e0455.png)

(This causes it to rebuild the image when the base node image changes, so we get security updates automatically)